lsp/kurtosis_extension.go

```diff
@@ -0,0 +1,179 @@
+package lsp
```

Comment:
What is the rationale of having this as a separate go module?

Reply:
The main rationale here is that I am following `Single Responsibility Principle` - everything related to kurtosis apis is part of this package. This separation can easily allow us to improve the plugin suggestions without changing the code much, and importantly once we have an API ready to mostly need to change this package and everything else should largely remain the same. ( Open/Closed principle) @victorcolombo

Reply:
Is it cool if I make that change in the next PR? @victorcolombo .. This will just allow for folks to do quick testing without relying on any manual steps? @victorcolombo